### PR TITLE
7ztm: Add version 2.1.1

### PR DESCRIPTION
**Pull request by ClassicDarkChocolate**
[7zTM](http://www.7ztm.de/) is a theme manager for 7-Zip.

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/7ztm.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "2.1.1",
+    "description": "Theme manager for 7-Zip",
+    "homepage": "http://www.7ztm.de/",
+    "license": "Freeware",
```
Comment (issaclin32) [inline bucket/7ztm.json:5]:
```suggestion
    "license": {
        "identifier": "Freeware",
        "url": "http://www.7ztm.de/index.php?cat=01_English&page=01_FAQ"
    },
```

Comment (ClassicDarkChocolate) [reply bucket/7ztm.json]: Thanks!

**Comment by rashil2000** [comment]
checkver should be removed if the app is not getting any updates

**Comment by issaclin32** [comment]
By the way. [7-zip 9.23](https://www.7-zip.org/history.txt) was released on **2011-06-07**. We can safely remove *checkver* because *7zTM* has not been updated for a long time.

http://www.7ztm.de/index.php?cat=01_English&page=02_Download
```
Current version:
2.1
2.1.1 Hotfix for 7-Zip 9.23 alpha
```

**Review by ClassicDarkChocolate — COMMENTED** [review]
Remove checkver

**Review by issaclin32 — APPROVED** [review]

**Pull request merged**